applications/API/forms.py

```
@@ -0,0 +1,284 @@
+    """
+    Invalidate insult cache.
+
+    Args:
+        reason: Reason for invalidation
+    """
+    insult_choices_manager.invalidate_cache(reason)
+
+
+def get_cache_stats() -> dict:
+    """Get current cache statistics for insult data."""
+    return insult_choices_manager.get_cache_stats()
```

Ruff (0.12.2)

Undefined name `insult_choices_manager`

(F821)